src/main.c

```diff
@@ -2408,6 +2412,23 @@ void handle_mouse_input() {
     }
 }
 
+
+float setJump(int megaJump, float dy)
+{
+    if ((megaJump) && (dy == 0))
+    {
+        dy = 40;
+        
+    }
+    else if (dy == 0)
+    {
+        dy = 8;
+        
+    }
+    
+    return dy;
+}
+
```

Comment:
curly braces in front of if and else if statements should be formatted uniformly. Make all format of if and else if statements similar to  line 2254.